cves/kernel/CVE-2013-2058.yml

```diff
@@ -369,8 +241,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: \
+    The check could happen at any time, but needs to happen. 
```

Comment:
Not to be nit picky, but ideally it happens before it's sent across the network :)